src/marionnette_extensions/page.coffee

```diff
@@ -7,10 +7,11 @@ class Page extends Marionette.LayoutView
   canGoBack: ->
     @_parent.canGoBack()
 
-  transitionBack: (route, options) ->
+  transitionBack: (route, options = {}) ->
```

Comment:
I saw that line too, but the options hash send to goBack and goHome in AnimatableRegion never gets passed to te show method because the navigation is done with:

`window.location.hash = route` line 36